libs/cdk/src/lib/utils/directives/focusable-item/focusable-item.directive.ts

```diff
@@ -84,7 +84,7 @@ export class FocusableItemDirective implements HasElementRef {
     private _tabbable = true;
 
     /** @hidden */
-    private _timerId: number | null;
+    private _timerId: any | null;
```

Comment:
You can use https://www.typescriptlang.org/docs/handbook/utility-types.html#returntypetype:
```
private _timerId: ReturnType<typeof setTimeout> | null;
```

Reply:
thanks!